### PR TITLE
Update Privacy Policy to be backed by Contentful

### PR DESCRIPTION
**Pull request by jeffh**
This is intended to be merged on Wednesday – Sept 8th.

---

Static pages can be updated in Contentful. Add page: `/policy/privacy/v1` to preserve the old privacy page (which is also in Contentful)

Implementation details:
- Contentful pages are preferred over local static files (matched by path)
- To avoid excessive API calls to contentful, a list of valid paths that contentful has defined to know when to query against contentful. This is every 5 mins (60 mins in dev, since reset will also fetch).
- Slow fetch interval for contentful to reduce API usage when in dev (resets will still trigger API calls) to 60 min
- Add preview api key to support previewing unpublished contentful pages
- Accept query parameter for `?preview=` to a hardcoded value for now. Future work is to make this value private (and use secure equality). Without securing, this exposes draft pages.
- Contentful pages are accessed via GraphQL
- Added `postcss-inherit` to have css classes inherit other properties from other css classes (for markdown css styling)
- Add markdown parser
- Converts the tree of contentful data for a StaticPage into a hiccup-style html format for stringifcation.

Supported Contentful Models:
- Markdown
- Html
- Paragraph

Supported Contentful Data Types:
- Rich Text Field

Refactors:
 - Move at-at's pool to a component that is shared

Limitations:
- Paths must be still specified in storefront's routing before new pages can be visible from Contentful
- Code embed in rich text isn't supported
- No visual difference between inline / block embed of assets + entries.
- Paragraphs (rich text content modules) can only nest once. This is a limitation of GraphQL without resorting to recursive calls.

### CODE REVIEW

File: src/storefront/config.clj [hunk 1]
```diff
@@ -83,18 +83,32 @@
...
+                                         :env-id                     "master"}
                      :logging           {:system-name "storefront.system"}})
 
 (defn env-config []
   {:environment         (env :environment)
    :bugsnag-token       (env :bugsnag-token)
    :welcome-config      {:url (env :welcome-url)}
-   :contentful-config   {:api-key  (env :contentful-content-delivery-api-key)
-                         :space-id (env :contentful-space-id)}
+   :contentful-config   {:cache-timeout              (* (if-dev-else env 4 60) minute)
```
Comment (tophat8855) [inline src/storefront/config.clj:107]: Hi, it seems to me that the dev and prod values are flipped here- should it be `(if-dev-else env 60 4)` and the same for the line below.

Comment (jeffh) [reply src/storefront/config.clj:107]: Thanks, I'll fix that.

File: src/storefront/config.clj [hunk 2]
```diff
@@ -83,18 +83,32 @@
...
+                                         :env-id                     "master"}
                      :logging           {:system-name "storefront.system"}})
 
 (defn env-config []
   {:environment         (env :environment)
    :bugsnag-token       (env :bugsnag-token)
    :welcome-config      {:url (env :welcome-url)}
-   :contentful-config   {:api-key  (env :contentful-content-delivery-api-key)
-                         :space-id (env :contentful-space-id)}
+   :contentful-config   {:cache-timeout              (* (if-dev-else env 60 4) minute)
```
Comment (tophat8855) [inline src/storefront/config.clj:107]: Hi- we think the 60 and the 4 should be switched.

**Review by tophat8855 — CHANGES_REQUESTED** [review]
We added a comment to the

File: src/storefront/config.clj (hunk 2 above)
Comment (jeffh) [reply src/storefront/config.clj:107]: Is this https://github.com/Mayvenn/storefront/pull/119/commits/fb0d27f0d1bee44e4f98c5cc413c158016413d98 not the fix?

Comment (tophat8855) [reply src/storefront/config.clj:107]: whoops, all is good.

**Review by tophat8855 — APPROVED** [review]

**Pull request merged**